daq/runner.py

```diff
@@ -211,7 +211,10 @@ def _init_daq_run_id(self):
     def _init_device_result_client(self):
         server_port = self.config.get('device_reporting', {}).get('server_port')
         if server_port:
-            return DeviceReportClient(server_port=server_port)
+            timeout = self.config['device_reporting'].get('rpc_timeout_sec')
+            if timeout:
+                return DeviceReportClient(server_port=server_port, rpc_timeout_sec=timeout)
```

Comment:
NB: I'm surprised Python doesn't have a more elegant way of doing this!  I've run into the same problem before...